libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/configuration/limitValues/shared/limitingItemsHook.ts

```diff
@@ -390,3 +401,28 @@ const parseCommonDateFilter = (
         isCommonDate: true,
     };
 };
+
+export const useCommonDateFilterTitle = (intl: IntlShape): string => {
```

Comment:
seems to be quite generic hook, is this file the right location for it?

Reply:
Moved to `DateFilter` folder structure.